frontend/src/component/admin/serviceAccounts/ServiceAccountsTable/ServiceAccountModal/ServiceAccountModal.tsx

```diff
@@ -225,16 +224,18 @@ export const ServiceAccountModal = ({
         !serviceAccounts?.some(
             (serviceAccount: IUser) => serviceAccount.username === value
         );
+    const isPATValid =
+        tokenGeneration === TokenGeneration.LATER ||
+        (isNotEmpty(patDescription) && patExpiresAt > new Date());
     const isValid =
         isNotEmpty(name) &&
         isNotEmpty(username) &&
         (editing || isUnique(username)) &&
-        (tokenGeneration === TokenGeneration.LATER ||
-            isNotEmpty(patDescription));
+        isPATValid;
 
     const suggestUsername = () => {
         if (isNotEmpty(name) && !isNotEmpty(username)) {
-            const normalizedFromName = `service:${name
+            const normalizedFromName = `service-${name
```

Comment:
After discussing with @NicolaeUnleash we changed the SA username auto suggestion to have a `service-` prefix instead of `service:`. 

Original idea was to provide an easy but optional way to distinguish SAs from regular users by username. We're totally open to suggestions here 👍 

![image](https://user-images.githubusercontent.com/14320932/210997064-36dd3c2f-7495-415f-994c-02b3f982c9f8.png)